app/views/api/integrations/apicast/configuration_driven/_configuration.html.slim

```diff
@@ -42,8 +72,9 @@
                                 placeholder: "https://api.#{parameterized_org_name_of_the_current_account}.com" },
                   hint: apicast_endpoint_input_hint(@service, environment: 'production')
         = help_bubble( 'proxy_endpoint_bubble', t('api.integrations.proxy.proxy_endpoint_help_html'))
-        div.help-button-constrainer
-          = render 'api/integrations/apicast/shared/mapping_rules', f: f
+        - unless @service.act_as_product?
```

Comment:
I'd say let's either use `- if @service.act_as_traditional_service?` everywhere instead of sometimes that and sometimes `- unless @service.act_as_product?` ooooor let's remove the method `act_as_traditional_service?` and use `- unless @service.act_as_product?` everywhere 😄